src/components/CTable.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="container pt-6 mx-auto">
+    <table class="table-auto mx-auto ">
```

Comment:
I really can't say much about the CSS as i'm not familiar with tailwind, this seems fine to me. just try to be consistent and avoid double spaces like in the tr or spaces at the end like here.